webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -22,8 +28,10 @@ class Layout extends React.Component {
       currentOrganization,
       changeActiveMenu,
       activeMenu,
+      onCollapse,
     } = this.props;
     if (items.length === 0) fetchMenuItems(data);
+    if (checkCollapsed()) onCollapse();
```

Comment:
I'm wondering...

Instead of calling `onCollapse` when we want to initiate the state with `isCollapse=true`.
Would it make sense to do it in the initial state creation inside the reducer?

```js
const initialState = Immutable({
  isCollapsed: checkCollapsed(),
});
```